pkg/logstash.sysv

```diff
@@ -96,12 +97,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      if [ $KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
+      if [ $LS_KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
```

Comment:
Sorry for the confusion, I agree with the quoting, I just failed to mention it here in comments, but it was in my latest push.  However, I still believe that using = rather than -eq is preferable because then no error is generated in any case.